docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -4,8 +4,68 @@ sidebar_position: 1
 
 # Creating Service Binding
 
-In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
-- Exposing binding-metadatas by default
-If it's not the case, please refer to the following section(#)
+In this section, we consider that the service you want to connect to is already
```

Comment:
We should provide a link to the other section of the documentation which explains how to do that.